### PR TITLE
docs: add organization integration doc

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

provide general tech implementation guide

### CODE REVIEW

File: docs/docs/recipes/organizations/README.mdx [hunk 1]
```diff
@@ -6,11 +6,11 @@ import Availability from '@components/Availability';
 
 # 🏢 Organizations (Multi-tenancy)
 
-<Availability cloud="comingSoon" oss={{ major: 1, minor: 12 }} />
+<Availability cloud="comingSoon" oss="comingSoon" />
```
Comment (xiaoyijun) [inline docs/docs/recipes/organizations/README.mdx:9]: Should be ready now?

File: docs/docs/recipes/organizations/impact-on-end-users.mdx [hunk 2]
```diff
@@ -8,4 +8,4 @@ Currently, there's no impact to the end-user sign-in experience since Logto deco
 
 As the user signs in, they can get the access to all organizations with membership. For detailed information, please refer to Organization RBAC.
 
-We may add advanced features for organizations that change the sign-in experience in the future. Please
+We may add advanced features for organizations that change the sign-in experience in the future. Stay tuned! Join our [Discord community](https://discord.gg/UEPaF3j5e6) to share your thoughts and feedback.
```
Comment (xiaoyijun) [inline docs/docs/recipes/organizations/impact-on-end-users.mdx:11]:
```suggestion
We may add advanced features for organizations that will change the sign-in experience in the future. Stay tuned! Join our [Discord community](https://discord.gg/UEPaF3j5e6) to share your thoughts and feedback.
```

File: docs/docs/recipes/organizations/integration.mdx [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+
+  </TabItem>
+
+  <TabItem value="others" label="Others">
+
+```ts
+const config = {
+  // ...
+  scope: 'openid offline_access urn:logto:scope:organizations',
+  resource: 'urn:logto:resource:organizations',
```
Comment (xiaoyijun) [inline docs/docs/recipes/organizations/integration.mdx:134]: When using SDKs, the `resource` will be added automatically? Maybe we should point it out.

File: docs/docs/recipes/organizations/README.mdx (hunk 1 above)
Comment (gao-sun) [reply docs/docs/recipes/organizations/README.mdx:9]: no. will update later

File: docs/docs/recipes/organizations/integration.mdx (hunk 3 above)
Comment (gao-sun) [reply docs/docs/recipes/organizations/integration.mdx:134]: image

this?

File: docs/docs/recipes/organizations/impact-on-end-users.mdx (hunk 2 above)
Comment (gao-sun) [reply docs/docs/recipes/organizations/impact-on-end-users.mdx:11]: "change"  is for "features". "in the feature" modifies "add".

**Review by charIeszhao — APPROVED** [review]
LGTM

**Pull request merged**